kgforge/core/archetypes/store.py

```diff
@@ -127,28 +128,38 @@ def retrieve(self, id: str, version: Optional[Union[int, str]]) -> Resource:
         # TODO These two operations might be abstracted here when other stores will be implemented.
         pass
 
+    def _retrieve_filename(self, id: str) -> str:
+        # TODO This operation might be adapted if other file metadata are needed.
+        not_supported()
+
     def download(self, data: Union[Resource, List[Resource]], follow: str, path: str) -> None:
```

Comment:
It would be nice to introduce a choice for a user to still erase if his/her intent is to only have the latest download.

Reply:
Should it be done in this PR?

Reply:
In this one or in another one.

Reply:
Added in this PR as commit 4a54135.